manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer Desktop
+License: © fjsoft.at
+ShortDescription: Verbinden Sie Ihr Handy über WLAN, USB-Kabel oder Bluetooth mit Ihrem PC und Sie werden überrascht sein, wie einfach und effizient Sie Ihr Smartphone mit diesem kompakten Programm verwalten können!
+ManifestType: defaultLocale

```

```suggestion
ManifestType: locale
```
Please use English as the default locale.